plugins/source_test.go

```diff
@@ -322,59 +297,6 @@ func testSyncTable(t *testing.T, tc syncTestCase) {
 	}
 }
 
-func TestTablesForSpec(t *testing.T) {
```

Comment:
Can we bring back this test in some form? Otherwise there is no test for `TablesForSpec`

Reply:
I think this is already tested under `serve` so not sure there is need to test it twice. most tests should go under `serve` given our plugin is called only via `grpc`

Reply:
Ah okay, yeah makes sense